.github/.github/dependabot.yml

```diff
@@ -0,0 +1,16 @@
+version: 2
+updates:
+  - package-ecosystem: "npm"
```

Comment:
Shouldn't this be Yarn?

Reply:
Both `yarn` and `npm` use value `npm` https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#package-ecosystem